lib/routes/ehentai/ehapi.js

```diff
@@ -35,7 +51,14 @@ async function parsePage(cache, data, get_bittorrent = false) {
         const el_a = el.find('td[class="gl3c glname"] a');
         const el_img = el.find('td.gl2c div.glthumb div img');
```

Comment:
The current selector will match "Front Page Settings" of "Compact" only.
```diff
-        const el_a = el.find('td[class="gl3c glname"] a');
-        const el_img = el.find('td.gl2c div.glthumb div img');
+        const el_a = el.find('td[class^="gl3"] a');
+        const el_img = el.find('td[class^=gl2] div.glthumb div img');
```
This will match "Minimal", "Minimal+" and "Compact". It would be better if you can make it match all 5 "Front Page Settings".